projects/media-viewer/src/lib/store/effects/bookmarks.effects.spec.ts

```diff
@@ -1,100 +1,90 @@
     TestBed.configureTestingModule({
-      providers: [provideMockActions(() => mockActions$)],
+      imports: [HttpClientTestingModule],
+      providers: [
+        {
+          provide: BookmarksApiService,
+          useValue: UserServiceMock,
+        },
+        BookmarksEffects,
+        provideMockActions(() => actions$)
+      ]
     });
-  })
 
-  it('should trigger load bookmarks success',  fakeAsync(() => {
-    mockActions$ = of(new LoadBookmarks('url'))
-    effects = new BookmarksEffects(mockActions$, mockApiService);
-    const mockRsp = { body: [bookmark], status: 200 };
-    spyOn(mockApiService, 'getBookmarks').and.returnValue(of(mockRsp))
-
-    effects.loadBookmarks$.subscribe(action => expectedAction = action);
-    tick();
-
-    expect(expectedAction).toEqual(new LoadBookmarksSuccess(mockRsp));
-  }));
-
-  it('should trigger load bookmarks failure', fakeAsync(() => {
-    mockActions$ = of(new LoadBookmarks('url'))
-    effects = new BookmarksEffects(mockActions$, mockApiService);
-    spyOn(mockApiService, 'getBookmarks').and.returnValue(throwError('error'))
-
-    effects.loadBookmarks$.subscribe(action => expectedAction = action );
-    tick();
-
-    expect(expectedAction).toEqual(new LoadBookmarksFailure('error' as any));
-  }));
-
-  it('should trigger create bookmarks success',  fakeAsync(() => {
-    mockActions$ = of(new CreateBookmark(bookmark))
-    effects = new BookmarksEffects(mockActions$, mockApiService);
-    spyOn(mockApiService, 'createBookmark').and.returnValue(of(bookmark))
-
-    effects.createBookmark$.subscribe(action => expectedAction = action);
-    tick();
-
-    expect(expectedAction).toEqual(new CreateBookmarkSuccess(bookmark));
-  }));
-
-  it('should trigger create bookmarks failure', fakeAsync(() => {
-    mockActions$ = of(new CreateBookmark(bookmark))
-    effects = new BookmarksEffects(mockActions$, mockApiService);
-    spyOn(mockApiService, 'createBookmark').and.returnValue(throwError('error'))
-
-    effects.createBookmark$.subscribe(action => expectedAction = action );
-    tick();
-
-    expect(expectedAction).toEqual(new CreateBookmarkFailure('error' as any));
-  }));
-
-  it('should trigger delete bookmarks success',  fakeAsync(() => {
-    mockActions$ = of(new DeleteBookmark('bookmarkId'))
-    effects = new BookmarksEffects(mockActions$, mockApiService);
-    spyOn(mockApiService, 'deleteBookmark').and.returnValue(of('bookmarkId'));
-
-    effects.deleteBookmark$.subscribe(action => expectedAction = action);
-    tick();
-
-    expect(expectedAction).toEqual(new DeleteBookmarkSuccess('bookmarkId'));
-  }));
-
-  it('should trigger delete bookmarks failure', fakeAsync(() => {
-    mockActions$ = of(new DeleteBookmark('bookmarkId'))
-    effects = new BookmarksEffects(mockActions$, mockApiService);
-    spyOn(mockApiService, 'deleteBookmark').and.returnValue(throwError('error'))
+    effects = TestBed.get(BookmarksEffects);
+  });
+
+  describe('getBookmarks$', () => {
```

Comment:
why are there no failure tests, if we're removing them we need to replace them, otherwise we're just halving the coverage

Reply:
I just copied what was done in annotations.effect.spec.ts... I will add the failure tests now :)

Reply:
Done!